docs/pages/management/admin/uninstall-teleport.mdx

```diff
@@ -0,0 +1,411 @@
+  </TabItem>
+  <TabItem label="Windows" scope="enterprise">
+
+  Remove the `tsh.exe` binary from the machine:
+
+  ```code
+  $ del C:\Path\To\tsh.exe
+  ```
+
+  </TabItem>
+  </Tabs>
+  </TabItem>
+  <TabItem label="Cloud" scope="cloud">
+
+  <Tabs>
+  <TabItem label="Debian/Ubuntu Linux (DEB)" scope="cloud">
+
+  Uninstall the Teleport binary using APT:
+
+  ```code
+  $ sudo apt-get -y remove teleport-ent
+  # NOTE: Older Cloud users may be running OSS binaries instead
+  # $ sudo apt-get -y remove teleport
+  ```
+
+  Uninstall the Teleport APT repo:
+
+  ```code
+  $ sudo rm -f /etc/apt/sources.list.d/teleport.list
+  ```
+
+  <Admonition type="notice" title="Uninstall standalone DEB package">
+  If the commands above do not work, you may have installed Teleport using a standalone DEB package. Remove it with:
+
+  ```code
+  $ sudo dpkg -r teleport-ent
+  # NOTE: Older Cloud users may be running OSS binaries instead
+  # $ sudo dpkg -r teleport
+  ```
+  </Admonition>
+
+  </TabItem>
+  <TabItem label="Amazon Linux 2/RHEL (RPM)" scope="cloud">
+
+  Uninstall the Teleport binary using YUM:
+
+  ```code
+  $ sudo yum -y remove teleport-ent
+  # Optional: Use DNF on newer distributions
+  # $ sudo dnf -y remove teleport-ent
+  # NOTE: Older Cloud users may be running OSS binaries instead
+  # $ sudo yum -y remove teleport
+  # $ sudo dnf -y remove teleport
+  ```
+
+  Uninstall the Teleport YUM repo:
+
+  ```code
+  $ sudo rm -f /etc/yum.repos.d/teleport.repo
+  ```
+
+  <Admonition type="notice" title="Uninstall standalone RPM package">
+  If the commands above do not work, you may have installed Teleport using a standalone RPM package. Remove it with:
+
+  ```code
+  $ sudo rpm -e teleport-ent
+  # NOTE: Older Cloud users may be running OSS binaries instead
+  # $ sudo rpm -e teleport
+  ```
+  </Admonition>
+
+  </TabItem>
+  <TabItem label="Linux Tarball" scope="cloud">
+
+  Remove the Teleport binaries from the machine:
+
+  ```code
+  $ sudo rm -f /usr/local/bin/tbot
+  $ sudo rm -f /usr/local/bin/tctl
+  $ sudo rm -f /usr/local/bin/teleport
+  $ sudo rm -f /usr/local/bin/tsh
+  ```
+
+  </TabItem>
+  <TabItem label="MacOS" scope="cloud">
+
+  Remove the Teleport binaries from the machine:
+
+  ```code
+  $ sudo rm -f /usr/local/bin/tbot
+  $ sudo rm -f /usr/local/bin/tctl
+  $ sudo rm -f /usr/local/bin/teleport
+  $ sudo rm -f /usr/local/bin/tsh
+  ```
+
+  <Admonition type="notice" title="Uninstall MacOS client tools">
+  If you installed the MacOS `tsh` client-only package and/or Teleport Connect for MacOS, you can optionally remove those too:
+
+  ```code
+  $ sudo rm -f /Applications/tsh.app
+  $ sudo rm -f /Applications/Teleport\ Connect.app
+  ```
+  </Admonition>
+
+  </TabItem>
+  <TabItem label="Windows" scope="cloud">
+
+  Remove the `tsh.exe` binary from the machine:
+
+  ```code
+  $ del C:\Path\To\tsh.exe
+  ```
+
+  </TabItem>
+  </Tabs>
+  </TabItem>
+</Tabs>
+
+## Step 3/3. Remove Teleport data and configuration files
```

Comment:
I wondered about this too, but I was a little wary about us changing the locations in code (especially for things like Teleport Connect) and not updating the docs. It might be useful though.

Reply:
I've added some clarifications.